frontend/src/components/status/SystemCheck.jsx

```diff
@@ -47,25 +24,25 @@ class SystemCheck extends React.PureComponent {
     const { description, isPassing, currentSettingsDetails } = this.props;
 
     return (
-      <div style={styles.container}>
-        <div style={styles.icon}>
+      <tr>
+        <td>{description}</td>
+        <td>
           {isPassing && (
             <div style={styles.passing}>
-              <Glyphicon style={styles.glyphicon} glyph="ok-circle" />
-              <Label>{LOCALIZE.commons.passStatus}</Label>
-              <p style={styles.currentSpecs}>{currentSettingsDetails}</p>
+              <i style={styles.glyphicon} className="fas fa-check-circle" />
+              <span>{LOCALIZE.commons.passStatus}</span>
+              <span style={styles.currentSpecs}>{currentSettingsDetails}</span>
             </div>
           )}
           {!isPassing && (
             <div style={styles.failing}>
-              <Glyphicon style={styles.glyphicon} glyph="remove-circle" />
-              <Label>{LOCALIZE.commons.failStatus}</Label>
-              <p style={styles.currentSpecs}>{currentSettingsDetails}</p>
+              <i style={styles.glyphicon} className="fas fa-times-circle" />
+              <span>{LOCALIZE.commons.failStatus}</span>
+              <span style={styles.currentSpecs}>{currentSettingsDetails}</span>
```

Comment:
Same comment as my previous one, but for the fail status:

Can we add a space between _**< span >**_ and _**{LOCALIZE.commons.failStatus}**_ and between _**< span style={styles.currentSpecs} >**_ and _**{currentSettingsDetails}**_, to have a little spaces between the check mark, the Pass word and the settings details?